go.mod

```diff
@@ -46,6 +46,7 @@ require (
 	github.com/stretchr/testify v1.7.0
 	github.com/tendermint/spn v0.0.0-20210406123257-decaff8dcaf9
 	github.com/tendermint/tendermint v0.34.8
+	github.com/tendermint/vue v0.0.0-20210407103210-b92631502f18
```

Comment:
@clockworkgr > b92631502f18

Reply:
ah.. ok...so specific commit.

Can it be configured to pull "latest" version from master instead once linting issue is fixed? That wil ensure we can update independently if needed

Reply:
Branches won't work with Go. The versions we use either must be tagged or only a spesific hash can be used.

Reply:
Ah so there is no @latest specifier or something...shame...

Reply:
We should be able to run `go get github.com/tendermint/vue@develop` to get `go.mod` updated with the latest version, not sure if it helps.